arch/arm/boot/dts/msm8974-v2.dtsi

```diff
@@ -123,7 +123,7 @@
 		<1880000 2068000>,
 		<3008000 3309000>,
 		<3760000 4136000>,
-		<4468000 2457000>;
+		<4468000 2457600>;
```

Comment:
As far as DTS files, isn't it vice versa? 14001 only goes up to 2.265ghz where as 15055 goes to 2.4ghz. I'm just curious in this regard. (msm8974pro.dtsi)